index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Green-on-Mind Design Designing Sustainably</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+        href="https://fonts.googleapis.com/css2?family=PT+Sans:wght@400;700&family=Sansita+Swashed:wght@600;800&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <nav>
+        <ul>
+            <li>
+                <a href="#details">details</a>
+
+            </li>
+            <li>
+                <a href="#speakers">speakers</a>
+            </li>
+            <li>
+                <a href="#register">register</a>
+            </li>
+        </ul>
+    </nav>
+
+    <h1>Green-on-Mind Design</h1>
+
+    <section id="details">
+        <h2>- details-</h2>
```

Comment:
As coded here, a screen reader would try to parse / understand these dashes. Two options:
- You can apply these lines via CSS and remove them from the markup; or
- You can wrap them in `span` elements and use ARIA attributes to tell screen readers to ignore them, like so:

```suggestion
        <h2><span aria-hidden="true">-</span> details <span aria-hidden="true">-</span></h2>
```

How you intend to style these dashes would play a role in deciding which option is best.